client/blocks/login/login-form.jsx

```diff
@@ -553,18 +554,16 @@ export class LoginForm extends Component {
 			return this.props.translate( 'Your username' );
 		}
 
-		return this.isPasswordView() ? (
-			this.renderChangeUsername()
```

Comment:
This was rendering a button in the label which doesn't work with the InputControl, because when the button is focused the InputControl considers that it has focus within and applies focus styles to the input.

I've changed this to either [render the button and a hidden label for screen readers if the username/email is set (`isPasswordView: true`)](https://github.com/Automattic/wp-calypso/pull/103319/files#diff-c4b202c48b075713a04477b1013b69e705758b4a0db59b3e648a48ed95077eddR860) or pass the text as the InputControl label prop if not.